src/Common/src/Interop/Windows/user32/Interop.GetWindowText.cs

```diff
@@ -10,7 +10,7 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32)]
+        [DllImport(Libraries.User32, EntryPoint = "GetWindowTextW")]
```

Comment:
There are ~80 odd places we pinvoke explicitly to W. Should we audit them for CharSet?